web-runtime/npm-package/src/copyMarine.ts

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
```

Comment:
Let's add a comment why this file is needed.

Reply:
[AquaVM](https://github.com/fluencelabs/aquavm/blob/master/avm/client/src/copyAvm.ts) and [avm-runner-background](https://github.com/fluencelabs/avm-runner-background/blob/main/avm-runner-background/src/copyRunnerScript.ts) have similiar files in their repositories without comments. Pavel says that is needed to make browser applications work. Looks like it is a common practice in JS world, maybe we do not need a comment then?